delfin/drivers/hpe/hpe_msa/ssh_handler.py

```diff
@@ -0,0 +1,370 @@
+import six
+
+from oslo_log import log as logging
+from delfin import exception
+from delfin.common import constants, alert_util
+from delfin.drivers.utils.ssh_client import SSHPool
+from delfin.drivers.utils.tools import Tools
+
+try:
+    import xml.etree.cElementTree as ET
+except ImportError:
+    import xml.etree.cElementTree as ET
+
+LOG = logging.getLogger(__name__)
+
+
+class SSHHandler():
+
+    SECONDS_TO_MS = 1000
+
+    DISK_PHYSICAL_TYPE = {
+        'fc': constants.DiskPhysicalType.FC,
+        'SAS': constants.DiskPhysicalType.SAS
+    }
+
+    def __init__(self, **kwargs):
+        self.ssh_pool = SSHPool(**kwargs)
+
+    def login(self):
+        try:
+            result = self.ssh_pool.do_exec('show pools')
+            if 'is not a recognized command' in result:
+                raise exception.InvalidIpOrPort()
+        except Exception as e:
+            LOG.error("Failed to login msa storwize_svc %s" %
+                      (six.text_type(e)))
+            raise e
+
+    def get_storage(self, storage_id):
+        try:
+            result = self.ssh_pool.do_exec('show system')
+            list = self.analysisDataToList(result, 'system')
```

Comment:
`list` is a type name of python, please do not use this as a variable name

Reply:
The specification of Python has been relearned, and this problem has been changed. It will be updated next time